.github/workflows/handle.yml

```diff
@@ -0,0 +1,21 @@
+# Workflow that handles AOD requests.
+name: 'aod-handle'
+
+on:
+  pull_request_review:
+    types: ['submitted']
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}-aod-request-handle'
+  cancel-in-progress: true
+
+jobs:
+  handle:
```

Comment:
I am thinking we should add the file check in [here](https://github.com/abcxyz/access-on-demand/blob/main/.github/workflows/handle.yml#L59), for example, if hashFiles('iam.yaml', 'gcloud.yaml') != '', continue the go and AOD cli install and run.

Without this check there, an approve event on a feature PR will install AOD cli though it won't run it. That step is time consuming.

We can add the check here as well.

Reply:
mark fixed.